images/access-setup/content/bin/setup_compute.sh

```diff
@@ -176,6 +178,28 @@ tekton_chains_manifest(){
   printf "OK\n"
 }
 
+tekton_results_manifest(){
+  manifest="$manifests_dir/compute/tekton-results/tekton-results-secret.yaml"
```

Comment:
I would expect that manifest to be applied in https://github.com/openshift-pipelines/pipeline-service/blob/main/images/cluster-setup/bin/install.sh#L133

Reply:
added now.